Vagrantfile

```diff
@@ -11,7 +11,7 @@ Vagrant.configure("2") do |config|
   # VM, but users are encouraged to test this and make adjustments below (or file PRs)
   # if you find the VM lagging or unresponsive.
   config.vm.provider "virtualbox" do |v|
-    v.memory = 1536
+    v.memory = 8000
```

Comment:
Does this change need to be part of this PR?

Reply:
Not at all, I should actually have left that as it was, that's something I forgot to change. When testing, the loading between pages took a little too long so I thought giving the VM a bunch of space would make things faster on my end. It didn't change anything, and it slipped right by me. We can change it to the way it was before.

Reply:
Okay yeah, please change it back, thanks!